test-programs/sdk-test-program/programs/sdk-test/tests/test.rs

```diff
@@ -168,8 +171,18 @@ where
     )
     .unwrap();
 
+    let proof_result = ProofRpcResult {
+        root_indices: rpc_result.address_root_indices.clone(),
```

Comment:
can't we just use one `ProofRpcResult` type in light-client everywhere, why do we need two?

Reply:
We have only one `ProofRpcResult`, [its declared in `sdk`](https://github.com/Lightprotocol/light-protocol/blob/sergey/refactor_test_indexer/sdk/src/proof.rs#L37-L42).